src/libs/elektra/symbols.map

```diff
@@ -178,6 +176,12 @@ libelektraprivate_1.0 {
 	keyReplacePrefix;
 	ksFindHierarchy;
 
+	ksBelow;
+
+	elektraIsArrayPart;
+
+	backendsDivide;
```

Comment:
This shouldn't be in public API, not even in private part. It at least needs elektra prefix. (otherwise there might be a name clash at library loading).

Reply:
It's only here because of tests. I suggest we keep it for now and change it when implementing #4246. Until the next release it doesn't really matter.